### PR TITLE
feat: title change

### LINKED ISSUE #2009: View Appointment screen is showing translation key for title
## 🐛 Bug Report

The title show the translation key on the View Appointment Screen.

## To Reproduce

1. Create a new appointment
2. Navigate and view that appointment
3. See that the title of the screen is the translation key

## Expected behavior
It should be the correct title instead of the translation key

### PR DESCRIPTION
**Pull request by michaeldaigler**
Fixes #2009

**Changes proposed in this pull request:**

- changed title of appointment view

### CODE REVIEW

File: src/scheduling/appointments/view/ViewAppointment.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import useAddBreadcrumbs from '../../../breadcrumbs/useAddBreadcrumbs'
 
 const ViewAppointment = () => {
   const { t } = useTranslation()
-  useTitle(t('scheduling.appointments.viewAppointment'))
+  useTitle(t('Appointments'))
```
Comment (jackcmeyer) [inline src/scheduling/appointments/view/ViewAppointment.tsx:17]: the string that is passed to the translation function needs to be a key defined in the translation files.

I would expect that this PR would not change this line of code, but would rather add a key in the translation files that allow for scheduling.appointments.viewAppointment to map to the string View Appointment.

**Comment by tehKapa** [comment]
Hi @michaeldaigler, thank you for your submission, we really appreciate it! Like many open source projects, we ask that you sign our Contributor License Agreement before we can accept your contribution :)

**Comment by michaeldaigler** [comment]
Thank you. Where can I sign the Contributor License Agreement?

**Comment by jackcmeyer** [comment]
Thanks for the contribution @michaeldaigler! However, this bug was fixed via #2016. Closing this PR.

**Pull request closed without merging**